mopidy_spotify/library.py

```diff
@@ -63,20 +63,13 @@ def _lookup_album(self, sp_link):
     def _lookup_artist(self, sp_link):
         sp_artist = sp_link.as_artist()
         sp_artist_browser = sp_artist.browse(
-            type=spotify.ArtistBrowserType.NO_TRACKS)
+            type=spotify.ArtistBrowserType.FULL)
```

Comment:
> That can't be right. sp_artist.browse (sp_artistbrowse_create) returns albums

That returns an `ArtistBrowser`.

I omitted the park about a future release because we all know that's never going to happen.